tests/distributed/test_kvcache_comm.py

```diff
@@ -0,0 +1,41 @@
+"""Test the KV cache communication operators.
+
+Run `python test_kvcache_comm.py`.
```

Comment:
I'm guessing there's no CI infra or test running?

Reply:
Actually, they do have one. I will have to figure out how to add this test to the CI infra. Multiple things to figure out - how to use pytest with argparse for LLMEngine and setting up MSCCL++ environment in the CI infra.